test/util/util.go

```diff
@@ -680,12 +662,22 @@ func ExpectPodsFinalized(ctx context.Context, k8sClient client.Client, keys ...t
 	for _, key := range keys {
 		createdPod := &corev1.Pod{}
 		gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
-			g.Expect(k8sClient.Get(ctx, key, createdPod)).To(gomega.Succeed())
+			g.Expect(client.IgnoreNotFound(k8sClient.Get(ctx, key, createdPod))).To(gomega.Succeed())
 			g.Expect(createdPod.Finalizers).Should(gomega.BeEmpty(), "Expected pod to be finalized")
 		}, Timeout, Interval).Should(gomega.Succeed())
 	}
 }
 
+func ExpectWorkloadsFinalized(ctx context.Context, k8sClient client.Client, keys ...types.NamespacedName) {
+	for _, key := range keys {
+		createdPod := &corev1.Pod{}
```

Comment:
Shouldn't this Get the Workload as in `expectWorkloadFinalized` before?

Reply:
Thanks!